hack/new-iam-user.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+###############################################################################
+#
+# new-iam-user.sh
+#
+# Convenience script adding a new IAM user to an existing AWS account.
+#
+# WARNING: This script will return secrets!
+#
+###############################################################################
+
+usage(){
+    echo "sh new-iam-user.sh <group-name> <user-name>"
+    exit 0
+}
+
+if [ -z "$1" ]; then
+    usage
+fi
+
+if [ -z "$2" ]; then
+    usage
+fi
+
+GROUP=$1
+USER=$2
+
+aws iam create-group --group-name ${GROUP}
```

Comment:
So I added a check for the `aws` utility in `$PATH`

Unsure what you mean 

> Also should we fail if the aws user cannot run the command? We are not checking error exits

Last I checked.. all shell/bash scripts will exit if an internal command fails with a non zero exit. ..